packages/ui/src/ui/store/selectors/tables.ts

```diff
@@ -0,0 +1,4 @@
+import {RootState} from '../../store/reducers';
+import {initialState} from '../../store/reducers/tables';
+
+export const getTables = (state: RootState) => state?.tables || initialState;
```

Comment:
It's a little confusing. Why not process this in the store? We need to look at in what case "state?.tables" may not exist and why this is not processed in the action

Reply:
the code was not changed it is just rewritted with typescript

Reply:
fixed